internal/stubserver/stubserver.go

```diff
@@ -56,9 +56,11 @@ type StubServer struct {
 	testgrpc.TestServiceServer
 
 	// Customizable implementations of server handlers.
-	EmptyCallF      func(ctx context.Context, in *testpb.Empty) (*testpb.Empty, error)
-	UnaryCallF      func(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error)
-	FullDuplexCallF func(stream testgrpc.TestService_FullDuplexCallServer) error
+	EmptyCallF           func(ctx context.Context, in *testpb.Empty) (*testpb.Empty, error)
+	UnaryCallF           func(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error)
+	FullDuplexCallF      func(stream testgrpc.TestService_FullDuplexCallServer) error
+	StreamingInputCallF  func(stream testgrpc.TestService_StreamingInputCallServer) error                                          // Client-Streaming request
+	StreamingOutputCallF func(req *testpb.StreamingOutputCallRequest, stream testgrpc.TestService_StreamingOutputCallServer) error // Server-streaming response
```

Comment:
Nit: Please remove the trailing comments for the newly added fields since the other existing fields don't have them *and* there is a top-level comment saying these are customizable implementations of server handlers, which is explanatory enough I feel. Thanks.